flink-connector-postgres-cdc/src/main/java/com/ververica/cdc/connectors/postgres/table/PostgreSQLTableFactory.java

```diff
@@ -187,6 +188,10 @@ public DynamicTableSource createDynamicTableSource(DynamicTableFactory.Context c
             validateIntegerOption(JdbcSourceOptions.CONNECT_MAX_RETRIES, connectMaxRetries, 0);
             validateDistributionFactorUpper(distributionFactorUpper);
             validateDistributionFactorLower(distributionFactorLower);
+        } else {
+            checkState(
+                    !StartupMode.LATEST_OFFSET.equals(startupOptions.startupMode),
+                    "The option 'scan.startup.mode' must not be 'latest-offset' when 'scan.incremental.snapshot.enabled' is not enabled.");
```

Comment:
"The Postgres CDC connector does not support  'latest-offset' startup mode when 'scan.incremental.snapshot.enabled' is disabled, you can enable 'scan.incremental.snapshot.enabled' to use startup mode.");